tools/acncli/deployment/mdnc_azure-cni-manager.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: azure-cni-manager
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      acn: azure-cni-manager
+  template:
+    metadata:
+      labels:
+        acn: azure-cni-manager
+    spec:
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
+      tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
+          operator: Exists
+      hostNetwork: true
+      containers:
+        - name: azure-cni-installer
+          image: mcr.microsoft.com/containernetworking/azure-cni-manager:v1.4.35
```

Comment:
confirmed this image exists? at some point we probably stopped building it

Reply:
I do see this release: https://github.com/Azure/azure-container-networking/releases/tag/v1.4.35 and that's also in our customer logs... let's see if there's any issue with my E2E run.

Reply:
Actually trying to pull the image manually it seems not found